### ISSUE TITLE
Directory including whitespace cannot used in SourceKit

### ISSUE THREAD

**Issue by d-date** [peripheryapp/periphery#76]
Note: After removing whitespace from directory, below error has resolved.

```
Error: SourceKit index request failed for file '/Users/d_date/dev/***/App/Screens/Item Detail/ItemDetailState.swift': '/Users/d_date/dev/***/App/Screens/Item Detail/ItemDetailState.swift' is not part of the input files.
```

**Comment by Datoolos** [comment]
I also encountered this issue, but it seems that it is fixed with revision 0d20682.

**Comment by Datoolos** [comment]
I am also facing this issue.
